definitions/types/gcp-security-iam.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "title": "IAM",
+  "description": "IAM Roles And Conditions",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^.*$": {
+      "type": "object",
+      "required": [
+        "role"
+      ],
+      "properties": {
+        "role": {
+          "title": "Role",
+          "description": "GCP Role",
+          "pattern": "^roles\/.*$",
```

Comment:
Might want to do `[a-zA-Z]+` instead of `.*` since I think that is a bit stricter, but more accurate (can add hyphens and underscores if GCP allows them).

Reply:
Good call, I added that pattern (plus dots) and some examples.